copy/observatory/scoring.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Scoring Methodology
+---
+
+# HTTP Observatory Scoring Methodology
+
+It is difficult to assign an objective value to a subjective question such as
+"How bad is not implementing HTTP Strict Transport Security?" In addition, what
+may be unnecessary for one site — such as implementing Content Security Policy —
+might mitigate important risks for another. The scores and grades offered by the
+Mozilla Observatory are designed to alert developers when they're not taking
+advantage of the latest web security features. Individual developers will need
+to determine which ones are appropriate for their sites.
+
+This page outlines the scoring methodology and grading system Observatory uses,
+before listing all of the specific tests along with their score modifiers.
+
+## Scoring Methodology
```

Comment:
This h2 overlaps with the h1, so maybe the h1 should indeed be "Scoring and Grading"?